CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.1.0](https://github.com/silinternational/ui-components/releases/tag/v1.1.0) - 2021-05-26
```

Comment:
That's different. We usually put them at the end of the file. (see https://raw.githubusercontent.com/silinternational/idp-pw-api/develop/CHANGELOG.md) Also, I usually see the link as the compare URL, but this works too -- probably better, actually.

Reply:
I wondered why your comment was at the bottom of the file.